templates/skeleton/TEMPLATE_GUIDELINES.md

```diff
@@ -0,0 +1,251 @@
+# Hydrogen Template Guidelines
+
+The following code guidelines are our opinionated best practices for developing templates in Remix and Hydrogen, with the goal of building consistent and well structured code.
```

Comment:
Small suggestion, I think we could combine these paragraphs and be slightly more direct. I'm not a great writer, so feel free to take this, adapt it or disregard it entirely.

```suggestion
This initial document provides guidelines and opinionated best practises for developing in Remix and Hydrogen with the goal of building consistent and well structured code. We hope this will evolve into more official recommendations, ESLint presets and other tooling to guide users to the outcomes defined below.
```

Reply:
```suggestion
This initial document provides guidelines and opinionated best practices for developing in Remix and Hydrogen with the goal of building consistent and well structured code. This may evolve into more official recommendations, such as ESLint presets and other tooling to guide users to the outcomes defined below.
```